composer.json

```diff
@@ -90,9 +90,6 @@
     ],
     "config": {
         "optimize-autoloader": true,
-        "sort-packages": true,
-        "platform": {
-            "php": "7.1.3"
```

Comment:
müssen wir irgendwo sicherstellen dass im release am ende nur dependencies drinnen sind, die auch mit 7.1.3 aka php-rex-min-version kompatibel sind?

Reply:
Die root-composer.json ist für die Releases irrelevant, in den anderen steht der Platform-Eintrag noch drin.